source/_integrations/trafikverket_weatherstation.markdown

```diff
@@ -14,75 +15,25 @@ ha_platforms:
 
 Showing weather information provided by [Trafikverket](https://www.trafikverket.se/) weather stations in Sweden.
 
-Potential use cases:
+## Potential use cases:
 
 - Get weather data in general.
 - You live near a weather station and want to know the current weather conditions at home.
 - Setup automations for your car heating system. If the road is frozen along the way to work, you might want the car heating system to start earlier.
 
-## Configuration
+{% include integrations/config_flow.md %}
 
-To enable this sensor, add the following lines to your `configuration.yaml`.
-
-```yaml
-sensor:
-  - platform: trafikverket_weatherstation
-    name: Trafikverket Kungälv
-    api_key: YOUR_API_KEY
-    station: Kungälv
-    monitored_conditions:
-        - air_temp
-        - road_temp
-        - humidity
-        - precipitation
-        - wind_direction
-        - wind_direction_text
-        - wind_speed
-        - wind_speed_max
-        - precipitation_amount
-        - precipitation_amountname
-```
-
-{% configuration %}
-name:
-  description: Your unique name of the device in the frontend.
-  required: true
-  type: string
-api_key:
-  description: Your personal API key from Trafikverket.
-  required: true
-  type: string
-station:
-  description: Name of the weather station from Trafikverket.
-  required: true
-  type: string
-monitored_conditions:
-  description: Specify what measurement data to retrieve from the weather station.
-  required: true
-  type: list
-  keys:
-    air_temp:
-      description: Air temperature.
-    road_temp:
-      description: The temperature in the road.
-    humidity:
-      description: Relative humidity.
-    precipitation:
-      description: Type of precipitation (Swedish text).
-    wind_direction:
-      description: Wind direction in degrees.
-    wind_direction_text:
-      description: Rough wind direction in twelve variants (Swedish text).
-    wind_speed:
-      description: Average wind speed during the last 10 minutes.
-    wind_speed_max:
-      description: Maximum wind speed measured during the last 30 minutes.
-    precipitation_amount:
-      description: Amount of precipitation.
-    precipitation_amountname:
-      description: Amount of precipitation in thirteen variants (Swedish text).
-    
-{% endconfiguration %}
+## Monitored conditions can be one of many of
```

Comment:
Monitored conditions are no longer allowed, see:

<https://github.com/home-assistant/architecture/blob/master/adr/0003-monitor-condition-and-data-selectors.md>

Reply:
Hi. In the config flow the user is presented with all options in a user friendly manner so he can choose and doesn't need to know the options available. Should I just call it something different instead to make this clear?

Reply:
> In the config flow the user is presented with all options in a user friendly manner so he can choose and doesn't need to know the options available

That is no longer allowed.

> Should I just call it something different instead to make this clear?

No, just expose all available entities instead. Home Assistant provides options for users to enable/disable entities nowadays.

Reply:
Clear! I will take care

Reply:
Very clear and now done so also then PR https://github.com/home-assistant/core/pull/60664 needs to be approved. Will update this markdown accordingly